trezorctl

```diff
@@ -28,6 +28,8 @@ import json
 import base64
 import tempfile
 from trezorlib import types_pb2 as types
+import time
+import textwrap
```

Comment:
@romanz Was using it for ASCII Armor (read: will be using it again very soon) so I don't really think it's worth rebasing yet